plugins/catalog-backend-module-gerrit/api-report.md

```diff
@@ -0,0 +1,51 @@
+## API Report File for "@backstage/plugin-catalog-backend-module-gerrit"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Config } from '@backstage/config';
+import { EntityProvider } from '@backstage/plugin-catalog-backend';
+import { EntityProviderConnection } from '@backstage/plugin-catalog-backend';
+import { GerritIntegration } from '@backstage/integration';
+import { LocationSpec } from '@backstage/plugin-catalog-backend';
+import { Logger } from 'winston';
+import { TaskRunner } from '@backstage/backend-tasks';
+
+// @public (undocumented)
+export class GerritEntityProvider implements EntityProvider {
+  constructor(
```

Comment:
You could consider making this `private`. More a choice, it would mean, you don't need to export the type `GerritProviderConfig`.